CommonTools/UtilAlgos/interface/EventSelectorAdapter.h

```diff
@@ -31,7 +31,7 @@ class EventSelectorAdapter : public edm::EDFilter
   virtual ~EventSelectorAdapter() {}
 
  private:
-  bool filter(edm::Event& evt, const edm::EventSetup& es) override { return eventSelector_(evt, es); }
+  bool filter(edm::StreamID, edm::Event& evt, const edm::EventSetup& es) const override { return eventSelector_(evt, es); }
```

Comment:
I highly advise to add ```override``` on all such interfaces to guarantee one doesn't make a typo.